tests/Person.spec.ts

```diff
@@ -441,4 +445,235 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
       await page.getByTestId("close-person-delete-success-dialog-button").click();
     });
   });
-});
+
+  test("Eine Lehrkraft anlegen in der Rolle Landesadmin und die Bestätigungsseite vollständig prüfen", async ({ page }) => {
+    const PersonCreationView = new PersonCreationViewPage(page);
+    const Rolle = "Lehrkraft";
+    const Vorname = "TAuto-PW-V-" + faker.person.firstName();
+    const Nachname = "TAuto-PW-N-" + faker.person.lastName();
+    const Schulstrukturknoten = "Testschule Schulportal";
+    const Dienstellennummer = "1111111";
```

Comment:
```suggestion
    const Dienststellennummer = "1111111";
```
Bitte alle Vorkommen auf Schreibfehler prüfen